### PR TITLE
leveraged profits fix

### PR DESCRIPTION
**Pull request by HojouFotytu**
- leverage wasn't being included in calculation for individual pair profits nor total bag gain, when using gainStrategy value

### CODE REVIEW

**Comment by HojouFotytu** [comment]
Logging every Threshold exception during SMS trigger checks is redundant, so I removed it.  The same information is already available to the user when building market trend averages.

**Comment by HojouFotytu** [comment]
Added an icon and tooltip next to SMS trends on the MarketAnalyzer page to indicate exclusions due to TrendThreshold.

**Comment by HojouFotytu** [comment]
Added a field for TrendThreshold to Settings Analyzer page.

**Review by JackTerok — APPROVED** [review]
Looks good

File: Core/Main/PTMagic.cs [hunk 1]
```diff
@@ -1648,11 +1648,6 @@ private void ApplySingleMarketSettings()
                                                   orderby m.Market
                                                   select m;
```
Comment (djbadders) [inline Core/Main/PTMagic.cs:1650]: Should remove lines 1642-1649 as well as we don't need the excludedMarkets if not outputting them to the log.

File: Core/Main/PTMagic.cs [hunk 2]
```diff
@@ -1862,11 +1857,6 @@ orderby m.Market
                                               orderby m.Market
                                               select m;
```
Comment (djbadders) [inline Core/Main/PTMagic.cs:1859]: Remove the code block to calculate excludedMarkets as no longer needed.

**Review by djbadders — CHANGES_REQUESTED** [review]
Looks good apart from we need to trim the excludedMarkets retrieval as we no longer use it.

**Comment by djbadders** [comment]
Looks good

**Pull request merged**